hail/src/main/c/Makefile

```diff
@@ -67,11 +68,6 @@ $(BUILD)/%.o: %.cpp
 	@mkdir -p $(@D)
 	$(CXX) -o $@ $(CXXFLAGS) -MD -MF $(@:%.o=%.d) -MT $@ -c $<
 
-# if libsimdpp does not exist (e.g. after a clean), UNIT64_VECTOR_SIZE needs to
-# be set explicitly via HAIL_OVERRIDE_WIDTH
-$(BUILD)/ibs.d: CXXFLAGS += -DHAIL_OVERRIDE_WIDTH=4 -DNUMBER_OF_GENOTYPES_PER_ROW=256
-$(BUILD)/ibs.o: CXXFLAGS += -DNUMBER_OF_GENOTYPES_PER_ROW=256
-
```

Comment:
The tests indicate this should just be set when we're compiling the functional-tests (the tests for ibs.cpp) so I moved it to the appropriate location down below.